### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by sstok**
|Q            |A  |
|---          |---|
|Bug Fix?     |yes|
|New Feature? |no |
|BC Breaks?   |no |
|Deprecations?|no |
|Fixed Tickets|   |
|License      |MIT|

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,8 +16,9 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "rollerworks/search": ">=1.0.0-beta4@dev",
-        "symfony/validator": "~2.3"
+        "rollerworks/search": ">1.0.0-beta4@dev",
+        "symfony/validator": "^2.3.3",
```
Comment (cordoval) [inline composer.json]: ^ is new to me, what is the meaning?

Comment (sstok) [inline composer.json]: Its something was added recently, `~1.0` means `>=1.0,

Comment (cordoval) [inline composer.json]: you meant ^1.0

Comment (sstok) [inline composer.json]: I have no idea :) this is how it used bot phpspec and it works.

`~2.3` is allowed. But I need at least 2.3.3 as anything lower has some serious bugs.

**Pull request merged**